src/components/Typography/Typography.types.ts

```diff
@@ -0,0 +1,76 @@
+import { TooltipProps } from 'antd'
+
+/**
+ * @link https://ant.design/components/typography#copyable
+ */
+export type CopyConfig = {
+
+  /**
+   * The text to copy.
+   */
+  text?: string;
+
+  /**
+   * Function called when the text is copied.
+   */
+  onCopy?: (event?: React.MouseEvent<HTMLDivElement>) => void;
+
+  /**
+   * Custom copy icon(s): [copyIcon, copiedIcon].
+   */
+  icon?: React.ReactNode;
+
+  /**
+   * Custom tooltip text(s): [copyText, copiedText]. If the attribute is set to false, no tooltip will be shown.
+   */
+  tooltips?: boolean | React.ReactNode;
+
+  /**
+   * The Mime Type of the text.
```

Comment:
Can We add this link here ? https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types